binding.cc

```diff
@@ -537,7 +423,7 @@ sn_crypto_generichash_batch(
   for (auto &buf : batch) {
     bool is_typedarray = false;
 
-    int err = js_is_typedarray(env, buf, &is_typedarray);
+    int err = js_is_typedarray(env, static_cast<js_handle_t &>(buf), is_typedarray);
```

Comment:
This check is unnecessary, the buffer is already declared as `js_typedarray_t<uint8_t>`.